gearcatalog/frontend/src/components/gear/EditGearForm.tsx

```diff
@@ -34,7 +35,7 @@ match: { params } }: EditGearProps) => {
     useInput(selectedGear.locking);
 
   useEffect(() => {
-    selectGear(params.id, token);
+    selectGear(params.id.toString(), token);
```

Comment:
I don't believe I ever got a resolution to my comment about converting IDs to strings on a previous PR. Why are you doing this?